### DISCUSSION TITLE
2η ΟΣΣ το Σάββατο 10/12 6μμ

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#55] [Announcements]
Κύριε Χωριανόπουλε, καλησπέρα σας.

Θα ήθελα να ρωτήσω αν τελικά η ώρα συνάντησής μας για αύριο Σάββατο 10/12 θα είναι στις 18:00. Επίσης, θα παρακαλούσα, αν συμφωνείτε κι εσείς, αύριο να μας δώσετε εν συντομία κάποιες επεξηγήσεις για τα ζητούμενα της 3ης ΓΕ.  Επειδή νομίζω ότι είναι δύσκολο να οργανωθεί μια συνάντηση τα επόμενα 4 Σαββατοκύριακα (17/12 υπάρχει τουλάχιστον μία ακόμη ΟΣΣ για ΘΕ του ΣΔΥ, 24/12, 31/12 και 7/1 πέφτουν σε μέρες γιορτών) καλό θα ήταν να είχαμε κάποιες οδηγίες για να μπορούμε να ξεκινήσουμε πριν τον Ιανουάριο και να μη χάνουμε το χρόνο μας πελαγοδρομώντας.  Το ξέρω ότι είναι λίγο πρώιμο (κι ελπίζω να μην αγχώνω τους συναδέλφους), αφού δεν έχουμε παραδώσει ακόμα την 2η ΓΕ, αλλά από πλευράς διαχείρισης θα το βρούμε μπροστά μας.

**Comment by epidrome** [comment]
αν συμφωνείτε όλοι προτείνω να αρχίσουμε στις 6μμ 

και να γίνει επιπλέον συμπληρωματική ΟΣΣ2 το επόμενο Σάββατο ίδια ώρα αφού υπάρχει ήδη παράταση μιας εβδομάδας.

σύνολο κάπου 8ώρες!!

**Comment by vkaragia** [comment]
Οκ από εμένα

**Comment by afoscolou** [comment]
Καλημέρα. Συμφωνώ.
